src/libraries/System.ComponentModel.TypeConverter/ref/System.ComponentModel.manual.cs

```diff
@@ -6,7 +6,11 @@ namespace System.ComponentModel.Design
     public abstract partial class DesignerOptionService
     {
         [System.ComponentModel.TypeConverter(typeof(DesignerOptionConverter))]
-        public sealed partial class DesignerOptionCollection { }
-        internal class DesignerOptionConverter { }
+        public sealed partial class DesignerOptionCollection
+        {
+        }
```

Comment:
Your search and replace logic should work with both styles. I don't think we should change our manual ref files just because the algorithm isn't smart enough.

Reply:
Yes i am gonna fix the regex to support this.